tests/Fixtures/Kernel.php

```diff
@@ -98,8 +103,22 @@ protected function configureContainer(ContainerBuilder $c, LoaderInterface $load
         if (\getenv('USE_FOUNDRY_BUNDLE')) {
             $foundryConfig = ['auto_refresh_proxies' => false];
 
-            if (\getenv('DATABASE_URL') && \getenv('USE_MIGRATIONS')) {
-                $foundryConfig['database_resetter'] = ['orm' => ['reset_mode' => 'migrate']];
+            if (\getenv('DATABASE_URL')) {
+                $foundryConfig['global_state'] = [
+                    TagStory::class,
+                    TagStoryAsInvokableService::class,
+                ];
+
+                if (\getenv('USE_MIGRATIONS')) {
+                    $foundryConfig['database_resetter'] = ['orm' => ['reset_mode' => 'migrate']];
+                }
+            }
+
+            if (\getenv('MONGO_URL') && !\getenv('USE_DAMA_DOCTRINE_TEST_BUNDLE')) {
+                $foundryConfig['global_state'] = [
+                    ODMTagStory::class,
+                    ODMTagStoryAsAService::class,
+                ];
```

Comment:
beside of testing stories with mongo, this way I can implicitly test:
- "standlone" story
- service story
- invokable service as story
without modifying underlying tests